content/tools/index.njk

```diff
@@ -1,24 +1,59 @@
 ---
 title: Tools
 banner: Open Source Tools
-sub: We build tools that millions of developers rely on
+sub: Support tools that you and million of developers rely on
 list_tag: OddTools
 action:
   text: Sponsor our open web contributions »
   url: https://opencollective.com/oddbird-open-source
+goal:
+  title: Support Logical Shorthands in CSS
+  author: miriam
+  sub: Can we get this process unstuck?
+  summary: |
+    The CSS Working Group recently resolved to add a size shorthand for setting
+    both the width and height of an element. Many people asked about using it to
+    set the ‘logical’ `inline-size` and `block-size` properties instead.
+    But ‘logical shorthands’ have been stalled in the working group for years.
+
+    With a little funding, we could make it happen!
+  url: /2025/03/20/logical-shorthand/
+  sponsor_url: https://opencollective.com/oddbird-open-source/contribute/css-logical-shorthands-86141
+  links: |
+    - **Demo:** [2-Value Logical Shorthands](https://codepen.io/miriamsuzanne/pen/RNwYrEB)
+    - **Progress:** Follow along in [Mia’s public notebook](https://css.oddbird.net/logical/research/)
+    - **Article:** [How do we move logical shorthands forward?](/2025/03/27/logical-options/)
```

Comment:
```suggestion
    - **Articles:** [Ongoing series on logical shorthands](/2025/03/27/logical-options/)
```

Reply:
adjusted in https://github.com/oddbird/oddleventy/pull/859/commits/59059a150691158f968c1d9230221cf6e60465b0